changelog.d/5582.change

```diff
@@ -0,0 +1 @@
+Move presence enabling to an offline mode user setting
```

Comment:
The file extension `change` is not correct and will be ignored by towncrier. Please refer to https://github.com/vector-im/element-android/blob/develop/CONTRIBUTING.md#changelog

In this case, I think it's best to use the `feature` extension. For the content, since it's more user facing, I will write "Add a setting to be able to always appear offline"